homework3/semi-uber/UI/src/app/app.module.ts

```diff
@@ -15,7 +16,12 @@ import { HomePageComponent } from "./home-page/home-page.component";
     LoginPageComponent,
```

Comment:
It is a good practice to group the components to separate modules and then just import them to the `app.module`. In this way you will not have to import hundreds of components into you `app.module` and leave the `declarations` section clean when your application will grow.